numpy/typing/tests/test_typing.py

```diff
@@ -373,9 +378,15 @@ def _test_reveal(
     lineno: int,
 ) -> None:
     """Error-reporting helper function for `test_reveal`."""
-    if reveal not in expected_reveal:
+    strip_pattern = re.pattern(r"(\w+\.)+(\w+)")
```

Comment:
```suggestion
    strip_pattern = re.compile(r"(\w+\.)+(\w+)")
```
Ah, the function is called `compile`, not `pattern`.

Reply:
thank you